### PR TITLE
[Doc] Add tools/libs to the init database steps

### LINKED ISSUE #11314: [Doc][pseudo cluster] Missing guide of add mysql jars to tools libs path
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Missing guide of add mysql jars to tools libs path

### Documentation Links

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

Fix #11314

### CODE REVIEW

**Review by EricGao888 — DISMISSED** [review]

**Review by zhongjiajie — DISMISSED** [review]

**Comment by zhongjiajie** [comment]
there is a deadlink found, could you please also fix the dead link in this PR, it related to the doc migration PR #11313 . You should change `/development/` to `docs/contribute`

**Comment by EricGao888** [comment]
> there is a deadlink found, could you please also fix the dead link in this PR, it related to the doc migration PR #11313 . You should change `/development/` to `docs/contribute`

I think we might have some doc checks in our `pre-commit hook` https://github.com/apache/dolphinscheduler/issues/11347 :thinking:

**Comment by SbloodyS** [comment]
> > there is a deadlink found, could you please also fix the dead link in this PR, it related to the doc migration PR #11313 . You should change `/development/` to `docs/contribute`
> 
> I think we might have some doc checks in our `pre-commit hook` #11347 🤔

I haven't used this new feature yet. 🤣 Can we add this to the docs too?

**Comment by EricGao888** [comment]
> > > there is a deadlink found, could you please also fix the dead link in this PR, it related to the doc migration PR #11313 . You should change `/development/` to `docs/contribute`
> > 
> > 
> > I think we might have some doc checks in our `pre-commit hook` #11347 🤔
> 
> I haven't used this new feature yet. 🤣 Can we add this to the docs too?

We have already had it in our docs. But later this week I will find a way to install the hook automatically for our developers in case they miss it in docs : ) https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/development-environment-setup.md#code-style

**Comment by zhongjiajie** [comment]
> > there is a deadlink found, could you please also fix the dead link in this PR, it is related to the doc migration PR #11313 . You should change `/development/` to `docs/contribute`
> 
> I think we might have some doc checks in our `pre-commit hook` #11347 🤔

It is not related to our CI like github action or pre-hook, this dead link because we separate our docs into multiple repos, but they have to connect to each other. In this case, we have to use the whole URL in our document, but when we change the place of source docs(PR1), the link will be changed, and a dead link error throw.
The reason why we can pass CI in PR1 is that before the PR1 merged, the whole URL link still work, but it become deadlink when PR1 is be merged

**Comment by EricGao888** [comment]
> > > there is a deadlink found, could you please also fix the dead link in this PR, it is related to the doc migration PR #11313 . You should change `/development/` to `docs/contribute`
> > 
> > 
> > I think we might have some doc checks in our `pre-commit hook` #11347 🤔
> 
> It is not related to our CI like github action or pre-hook, this dead link because we separate our docs into multiple repos, but they have to connect to each other. In this case, we have to use the whole URL in our document, but when we change the place of source docs(PR1), the link will be changed, and a dead link error throw. The reason why we can pass CI in PR1 is that before the PR1 merged, the whole URL link still work, but it become deadlink when PR1 is be merged

Got it, thx @zhongjiajie for the explanations!

File: README.md [hunk 1]
```diff
@@ -98,11 +98,11 @@ You are very welcome to communicate with the developers and users of Dolphin Sch
 
 ### Contributor over time
 
-[![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/dolphinscheduler)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=apache/dolphinscheduler)
+[![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=apache/dolphinscheduler)]
```
Comment (zhongjiajie) [inline README.md]: Maybe we should totally delete it, WDYT?
```suggestion
```

**Review by zhongjiajie — COMMENTED** [review]
other LGTM

File: README.md (hunk 1 above)
Comment (SbloodyS) [reply README.md]: Done.

**Review by zhongjiajie — APPROVED** [review]

**Review by EricGao888 — APPROVED** [review]

**Pull request merged**